apply.sh

```diff
@@ -1,2 +1,2 @@
-export DEFAULT_PROD_VERSION=1.4.0
+export DEFAULT_PROD_VERSION=1.5.0
```

Comment:
Just remove this file from your branch to resolve the merge conflict after rebasing/merging.